### PR TITLE
TN-2745 remove obsolete /explore page.

### PR DESCRIPTION
**Pull request by pbugni**

### CODE REVIEW

**Comment by mcjustin** [comment]
@pbugni  aren't there are a few other places where this is referred to?

`./portal/eproms/templates/eproms/404.html:                    <a href="/explore" class="tnth-splash-bottom-link"><span class="hidden-xs">{{ _("Explore How This Works") }}</span><span class="hidden-sm hidden-md hidden-lg">{{ _("Explore") }}</span></a><span class="tnth-splash-bottom-link separator"></span><a href="/about" class="tnth-splash-bottom-link"><span class="hidden-xs">{{ _("Learn About TrueNTH") }}</span><span class="hidden-sm hidden-md hidden-lg">{{ _("Learn") }}</span></a>

./portal/eproms/templates/eproms/500.html:                    <a href="/explore" class="tnth-splash-bottom-link"><span class="hidden-xs">{{ _("Explore How This Works") }}</span><span class="hidden-sm hidden-md hidden-lg">{{ _("Explore") }}</span></a><span class="tnth-splash-bottom-link separator"></span><a href="/about" class="tnth-splash-bottom-link"><span class="hidden-xs">{{ _("Learn About TrueNTH") }}</span><span class="hidden-sm hidden-md hidden-lg">{{ _("Learn") }}</span></a>

./portal/eproms/templates/eproms/portal.html:        <a class="btn-lg btn-tnth-primary" href="{{url_for('portal.explore')}}">{{ _("Learn More") }}</a>

./portal/static/less/eproms.less:  a[href*="explore"]{
`

**Comment by pbugni** [comment]
thank you @mcjustin  - i didn't realize those existed.  Adding @achen2401 to review what I yanked to confirm it's all still balanced.

File: portal/eproms/templates/eproms/404.html [hunk 1]
```diff
@@ -37,7 +37,7 @@ <h4>
                     <a href="/" class="btn feature-btn feature-btn-primary">{{ _("Back To Home") }}</a>
                 </div>
                 <div class="tnth-splash-bottom">
-                    <a href="/explore" class="tnth-splash-bottom-link"><span class="hidden-xs">{{ _("Explore How This Works") }}</span><span class="hidden-sm hidden-md hidden-lg">{{ _("Explore") }}</span></a><span class="tnth-splash-bottom-link separator"></span><a href="/about" class="tnth-splash-bottom-link"><span class="hidden-xs">{{ _("Learn About TrueNTH") }}</span><span class="hidden-sm hidden-md hidden-lg">{{ _("Learn") }}</span></a>
+                    <a href="/about" class="tnth-splash-bottom-link"><span class="hidden-xs">{{ _("Learn About TrueNTH") }}</span><span class="hidden-sm hidden-md hidden-lg">{{ _("Learn") }}</span></a>
```
Comment (achen2401) [inline portal/eproms/templates/eproms/404.html:40]: I am not sure if `Learn About Truenth` has been translated previously. We could use the same verbiage `Explore How This Works` to make sure no translation issue.

Comment (pbugni) [reply portal/eproms/templates/eproms/404.html:40]: I didn't add that - it was already there.  I expect it is already translated, that's all old data, right @achen2401 ?

Comment (achen2401) [reply portal/eproms/templates/eproms/404.html:40]: ah, thank you, paul. sorry I missed that.

**Pull request merged**